src/ChunkMap.cpp

```diff
@@ -1313,21 +1313,14 @@ void cChunkMap::DoExplosionAt(double a_ExplosionSize, double a_BlockX, double a_
 				a_Entity.TakeDamage(dtExplosion, nullptr, static_cast<int>((1 / std::max(1.0, DistanceFromExplosion.Length())) * 8 * ExplosionSizeInt), 0);
 			}
 
-			double Length = DistanceFromExplosion.Length();
-			if (Length <= ExplosionSizeInt)  // Entity is impacted by explosion
-			{
-				float EntityExposure = a_Entity.GetExplosionExposureRate(ExplosionPos, static_cast<float>(a_ExplosionSize));
-
-				// Exposure reduced by armor
-				EntityExposure = EntityExposure * (1.0f - a_Entity.GetEnchantmentBlastKnockbackReduction());
```

Comment:
Why remove `EntityExposure`?

Reply:
It seems to be broken (https://github.com/cuberite/cuberite/issues/4527), and I'm not sure how to fix it/how to incorporate it.

Reply:
`GetExplosionExposureRate` scales by the fraction of the entities bounding box that overlaps with the explosion, so should only be < 1 if the entity is on the boundary of the explosion.

You've also broken blast protection's effect on knockback.

Reply:
If you know this code better, then feel free to make contributions or take over this PR. I didn't really want to work on it, but I don't want to leave users exposed to this bug for too long either.